datamodel/changelogs/0001/51_aquifier_2015.sql

```diff
@@ -1,65 +1,65 @@
+ALTER TABLE tww_od.dss15_aquifer ADD COLUMN perimeter_geometry geometry('CURVEPOLYGON', :SRID);
+CREATE INDEX in_tww_od_dss15_aquifer_perimeter_geometry ON tww_od.dss15_aquifer USING gist (perimeter_geometry );
+COMMENT ON COLUMN tww_od.dss15_aquifer.perimeter_geometry IS 'Boundary points of the perimeter / Begrenzungspunkte der Fläche / Points de délimitation de la surface';
+ALTER TABLE tww_od.dss15_aquifer ADD COLUMN remark  varchar(80) ;
+COMMENT ON COLUMN tww_od.dss15_aquifer.remark IS 'General remarks / Allgemeine Bemerkungen / Remarques générales';
+ALTER TABLE tww_od.dss15_aquifer ADD COLUMN last_modification TIMESTAMP without time zone DEFAULT now();
+COMMENT ON COLUMN tww_od.dss15_aquifer.last_modification IS 'Last modification / Letzte_Aenderung / Derniere_modification: INTERLIS_1_DATE';
+ALTER TABLE tww_od.dss15_aquifer ADD COLUMN fk_dataowner varchar(16);
+COMMENT ON COLUMN tww_od.dss15_aquifer.fk_dataowner IS 'Foreignkey to Metaattribute dataowner (as an organisation) - this is the person or body who is allowed to delete, change or maintain this object / Metaattribut Datenherr ist diejenige Person oder Stelle, die berechtigt ist, diesen Datensatz zu löschen, zu ändern bzw. zu verwalten / Maître des données gestionnaire de données, qui est la personne ou l''organisation autorisée pour gérer, modifier ou supprimer les données de cette table/classe';
+ALTER TABLE tww_od.dss15_aquifer ADD COLUMN fk_provider varchar (16);
+COMMENT ON COLUMN tww_od.dss15_aquifer.fk_provider IS 'Foreignkey to Metaattribute provider (as an organisation) - this is the person or body who delivered the data / Metaattribut Datenlieferant ist diejenige Person oder Stelle, die die Daten geliefert hat / FOURNISSEUR DES DONNEES Organisation qui crée l’enregistrement de ces données ';
 -------
 CREATE TRIGGER
-update_last_modified_aquifier
+update_last_modified_dss15_aquifer
 BEFORE UPDATE OR INSERT ON
- tww_od._aquifier
+ tww_od.dss15_aquifer
 FOR EACH ROW EXECUTE PROCEDURE
  tww_sys.update_last_modified();
 
 -------
 
-/* ALTER TABLE tww_od.water_catchment ADD COLUMN fk_aquifier varchar (16);
-ALTER TABLE tww_od.water_catchment ADD CONSTRAINT rel_water_catchment_aquifier FOREIGN KEY (fk_aquifier) REFERENCES tww_od._aquifier(obj_id) ON UPDATE CASCADE ON DELETE set null; */
+/* ALTER TABLE tww_od.water_catchment ADD COLUMN fk_dss15_aquifer varchar (16);
+ALTER TABLE tww_od.water_catchment ADD CONSTRAINT rel_water_catchment_dss15_aquifer FOREIGN KEY (fk_dss15_aquifer) REFERENCES tww_od.dss15_aquifer(obj_id) ON UPDATE CASCADE ON DELETE set null; */
 
 
-/* ALTER TABLE tww_od.infiltration_installation ADD COLUMN fk_aquifier varchar (16);
-ALTER TABLE tww_od.infiltration_installation ADD CONSTRAINT rel_infiltration_installation_aquifier FOREIGN KEY (fk_aquifier) REFERENCES tww_od._aquifier(obj_id) ON UPDATE CASCADE ON DELETE set null; */
+/* ALTER TABLE tww_od.infiltration_installation ADD COLUMN fk_dss15_aquifer varchar (16);
```

Comment:
Also rename from fk_aquifier to fk_dss15_aquifier or not?

Reply:
Do you know why this is commented?